core/src/service/index.ts

```diff
@@ -248,13 +265,18 @@ export function enhanceAPI<T extends FN>(_apiFn: T, config?: APIConfig<T>) {
     },
     useData: (): PICK_DATA<T> | null => {
       const [data, setData] = React.useState(null);
+      const subscribeIdRef = React.useRef('');
```

Comment:
Prefer to use `_.uniqueId('id_')` as initial value.So not need `subscribeSeriesId` outside.